extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java

```diff
@@ -117,6 +118,10 @@ public static void beforeClass() {
     @AfterClass
     public static void afterClass() {
         KINESIS.shutdown();
```

Comment:
we also need to add null check on this var

Reply:
done